app/views/questions/_comment.html.erb

```diff
@@ -26,7 +26,10 @@
 
   </div>
   <div class="inline pull-xs-right" style="vertical-align: top">
-    <p><% if answer_id == 0 && current_user && (current_user.role == "admin" || current_user.role == "moderator" || comment.uid == current_user.uid)%>
+	<a rel="tooltip" title="Flag as spam" class="btn btn-sm btn-default btn-flag-spam-<%= comment.id %>" href="mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public+Lab&body=Hi,+I+found+this+comment+that+looks+like+spam+or+needs+to+be+moderated:+https://publiclab.org/<%= comment.parent.path %>#c<%= comment.cid %>+by+https://publiclab.org/profile/<%= comment.author.username %>+Thanks!">
+		<i class="fa fa-flag"></i>
+	</a>
+	<p><% if answer_id == 0 && current_user && (current_user.role == "admin" || current_user.role == "moderator" || comment.uid == current_user.uid)%>
```

Comment:
hi @Toiya , changes are looking great! Could you please attach a screenshot of the updated view? Thanks.

Reply:
I'm a little unsure if this is what you want but here are three screenshots of the flag button created, if I understand correctly. As you can see in the third one with the email, the words are currently seperated by plus-signs, which I guess is not what we want. I could try to fix this with my semi-limited HTML-skills, if you want to :)

![final-screenshot-flag-button](https://user-images.githubusercontent.com/13527966/40181033-74686f00-59e8-11e8-9c0f-7b3d08c8548e.png)

Reply:
You should be able to delete the existing db/schema.rb and replace it with a copy of the example file. Sorry that wasn't as clear as it could be!